src/hdf5_util_helpers.d.ts

```diff
@@ -18,25 +18,26 @@ export interface H5T_class_t {
 }
 
 export interface Metadata {
-    signed: boolean,
-    cset: number,
+    array_type?: ArrayTypeMetadata,
     compound_type?: CompoundTypeMetadata,
-    vlen: boolean,
+    cset: number,
     littleEndian: boolean,
-    type: number,
-    size: number,
     shape: Array<number>,
+    signed: boolean,
+    size: number,
     total_size: number,
-    array_type?: ArrayTypeMetadata,
+    type: number,
+    vlen: boolean,
```

Comment:
I reorganized the fields by alphabetic order so that they are easier to compare with test output.